tests/integration/integration.go

```diff
@@ -359,18 +359,18 @@ func K3sSaveLog(server *K3sServer, dump bool) error {
 	return nil
 }
 
-// Find the k3s binary and then use the command on kubectl
-func K3sKubectlCommand(inputArgs ...string) (string, error) {
-	k3sBin := findK3sExecutable()
-	k3sCmd := append([]string{"kubectl"}, inputArgs...)
-	cmd := exec.Command(k3sBin, k3sCmd...)
-	var out bytes.Buffer
-	cmd.Stdout = &out
-	err := cmd.Run()
+func ParseApiInfo() (string, error) {
+	client, err := k8sClient()
 	if err != nil {
-		return "", fmt.Errorf("%s", err)
+		return "", err
 	}
-	return out.String(), err
+
+	result, err := client.RESTClient().Get().AbsPath("/api").DoRaw(context.TODO())
```

Comment:
Does the Discovery API I suggested not work?

Reply:
Yep, I was receiving almost everything related to the node, but not the info about the api server address.

Reply:
OK, in that case what you want is:

```golang
  endpoints, err := client.CoreV1().Endpoints("default").Get(context.Background(), "kubernetes", v1.GetOptions{})
  if err != nil {
    return "", err
  }

  addresses := []string{}
  for _, subset := range endpoints.Subsets {
    for _, address := range subset.Addresses {
      addresses = append(addresses, address.IP)
    }
  }
  return strings.Join(addresses, ","), nil
```